redaxo/src/addons/mediapool/package.yml

```diff
@@ -15,7 +15,7 @@ page:
         media:     { title: translate:pool_file_list }
         upload:    { title: translate:pool_file_insert }
         structure: { title: translate:pool_cat_list,   perm: media/hasAll }
-        sync:      { title: translate:pool_sync_files, perm: media/hasAll }
+        sync:      { title: translate:pool_sync_files, perm: media[sync] }
```

Comment:
Bin bzgl perm nicht mehr so im thema.. muss das perm erst irgendwo registriert werden?

Reply:
Das findet hier in der package statt.